descriptions/api.github.com/api.github.com.yaml

```
@@ -4969,6 +4839,66 @@ paths:
         previews: []
         category: orgs
         subcategory: 
+  "/organizations/:organization_id/secret-scanning/alerts":
```

We don't have any id-based routes in the spec yet. Should this be 

```suggestion
  "/orgs/{org}/secret-scanning/alerts":
```

---

maybe add another lint rule that would prevent ID-based routes for orgs, repos, users for now?

---

This issue has been resolved @mattpage thanks!